knative-kubecon/serving/README.md

```diff
@@ -0,0 +1,19 @@
+After having applied the service from the build demo.
+
+1. Open `echo "http://$(minishift ip):9000/k8s/ns/myproject/pods"` to see your pods.
```

Comment:
Okay to do in a follow-up, I think it might help others repeat this demo if we have a sentence saying here that we're going to show things in the console.